### PR TITLE
github 101

### PR DESCRIPTION
**Pull request by jerrycowell96**

### CODE REVIEW

**Review by edgarriba — CHANGES_REQUESTED** [review]
@jerrycowell96 where is the file ? is this PR ready to review or should be draft ?

File: website/docs/support/github-101.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+git checkout -b "insert name of branch here"
+```
+
+First you have completed your code and you are ready to make a push, you should follow this workflow.
+
+
+Pre-commit your code
+```
+git add -p # Add the changes I want
+pre-commit run --all-files
```
Comment (edgarriba) [inline website/docs/support/github-101.md:68]: this is not really necessary if we do install pre-commit before. I'd add a previous step `pre-commit install`

File: website/docs/support/github-101.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+4. In the left sidebar, under  Personal access tokens, click Tokens (classic).
+
+5. Select Generate new token, then click Generate new token (classic).
+
+6. Give your token a descriptive name.
+
+:::tip
+To give your token an expiration, select the Expiration drop-down menu, then click a default or use the calendar picker.
+:::
+
```
Comment (edgarriba) [inline website/docs/support/github-101.md:35]:
```suggestion
```

File: website/docs/support/github-101.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+We use GitHub. There are all of our repositories and where you can access the files needed to being working on the project and Amiga.
+
+Prerequisites
+1) First you will need to have an Github account
+- To create on you can go [here](https://www.github.com)
+
+Once you do that you will need to generate a personal token:
+
+## Personal Access Token
```
Comment (edgarriba) [inline website/docs/support/github-101.md:18]:
```suggestion
## Personal Access Token

```

File: website/docs/support/github-101.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+title: GitHub 101
+---
+
+Welcome to GitHub 101!
+
+Here will be the guide to all the steps you need to follow in order to be able to participate in the challenge.
+
+We use GitHub. There are all of our repositories and where you can access the files needed to being working on the project and Amiga.
+
+Prerequisites
```
Comment (edgarriba) [inline website/docs/support/github-101.md:12]:
```suggestion
Prerequisites:

```

**Review by edgarriba — APPROVED** [review]
LGTM. Just minimum formatting changes and simplify the pre-commit steps

**Comment by Hackerman342** [comment]
@jerrycowell96 I'd recommend getting this updated and merged in!

**Comment by Hackerman342** [comment]
@guilhermedemouraa I've just updated these docs. Can you please give them a review to make sure they seem clear? Please merge if so.

File: website/docs/support/github-101.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+We use GitHub for all of our repositories and it is where you can access the files and code
+(hosted in repositories) you need to begin working on your project developing with the Amiga.
+
+## Create an account
+
+First you will need to have an Github account
+
+- To create one, just go to [github.com](https://www.github.com) and sign up!
+
+## Set up an ssh key
```
Comment (guilhermedemouraa) [inline website/docs/support/github-101.md:20]: I'd probably rename this section, as its content isn't necessarily just about creating an SSH key

**Review by guilhermedemouraa — APPROVED** [review]

**Review by edgarriba — APPROVED** [review]

**Pull request merged**